cluster/config-defaults.yaml

```diff
@@ -29,6 +29,14 @@ skipper_clusterratelimit: "false"
 skipper_clusterratelimit: "true"
 {{end}}
 
+# skipper backend timeout defaults
+skipper_expect_continue_timeout_backend: "30s"
+skipper_keepalive_backend: "30s"
+skipper_max_idle_connection_backend: 0
```

Comment:
I wonder if this is parsed correctly and doesn't end up as empty string or something like that. I would expect this should be a string to be correct but I'm not 100% sure. @aermakov-zalando do you know?

Reply:
`func Itoa(i int) string` should be fine in Go